### ISSUE TITLE
Support new emoji

### ISSUE THREAD

**Issue by jackiehluo** [arvida/emoji-cheat-sheet.com#367]
The last release of emoji added a LOT more that aren't listed anywhere on the Emoji Cheat Sheet. Would be really cool to get them into the list.

**Comment by JohnTian** [comment]
:tada:

**Comment by adam-lynch** [comment]
@jackiehluo do you mean iOS or what?

**Comment by jackiehluo** [comment]
I meant that the emoji from the Unicode 7.0 and 8.0 releases aren't listed—and we're about to get 9.0 soon, so it'll be even more out of date. Not sure what you mean by iOS. Emoji like :upside_down_face: or :lightning_cloud: aren't included right now in the cheat sheet.

**Comment by adam-lynch** [comment]
Thanks for clarifying @jackiehluo. I had no idea what you meant and assumed you meant Apple added new ones of their own in a recent iOS release or something. I'm curious which sources this project depends on and which sources emoji come from in general.

**Comment by Crissov** [comment]
@jackiehluo I think it’s safe to assume this project is either stale or dead. It adds these non-standard “emojis”:

:feelsgood:
:finnadie:
:goberserk:
:godmode:
:hurtrealbad:
:rage1:
:rage2:
:rage3:
:rage4:
:suspect:
:trollface:
:octocat:
:squirrel:
:shipit: 

Similar projects more alive:

* https://github.com/github/gemoji
* https://github.com/Ranks/emoji-alpha-codes/
* https://github.com/muan/emojilib

**Comment by adam-lynch** [comment]
@Crissov those emoji are listed in this project unless I'm misunderstanding. See `:suspect:` on http://www.webpagefx.com/tools/emoji-cheat-sheet/ for example. 

<img width="1224" alt="screen shot 2017-02-01 at 15 43 20" src="https://cloud.githubusercontent.com/assets/1427241/22513795/33908230-e895-11e6-9b84-b07662534e83.png">

**Comment by Crissov** [comment]
@adam-lynch Yes, these are the non-standard extensions supported by this project (and possibly others). The list of missing Unicode emojis is *much* longer, especially if one considers ZWJ sequences and skin tones.

**Comment by adam-lynch** [comment]
Ah sorry I misread. Thanks @Crissov
